.samples/dynamic-endpoint-passthrough.yaml

```diff
@@ -0,0 +1,12 @@
+displayName: Dynamic Endpoint Passthrough Service
+description: >-
+  Sends an HTTP GET to a Dynamic Endpoint(Endpoint URL is not known before hand) and responds with
```

Comment:
IMO we don't need to say `(Endpoint URL is not known before hand)` explicitly as the `Dynamic Endpoint` is self explanatory. WDYT?

Reply:
Addressed in https://github.com/wso2/choreo-samples/pull/155/commits/dae18a703dca4bcef0a1f06be5d5875f22cfc1a3